bindings/index.template.html

```diff
@@ -16,51 +16,70 @@
             ],
             edDraftURI: "https://w3c.github.io/wot-binding-templates/BINDINGNAME",
             github: "https://github.com/w3c/wot-binding-templates",// please do not change this
-            shortName: "wot-BINDINGNAME-template", // or "wot-BINDINGNAME-binding"
+            shortName: "wot-BINDINGNAME-binding",
             otherLinks: [{
-                key: "Core Binding Templates Specification",
+                key: "WoT Bindings Registry",
                 data: [
                     {
-                    value: "Web of Things (WoT) Binding Templates",
-                    href: "../../../index.html"
+                        value: "Web of Things (WoT) Bindings Registry",
+                        href: "https://w3c.github.io/wot-bindings-registry/"
                     }
                 ]
                 },
+                {
+                    key: "Bindings Section of the TD Specification",
+                    data: [
+                        {
+                            value: "Explanation of the binding mechanism",
+                            href: " https://w3c.github.io/wot-thing-description/#bindings"
+                        }
+                    ]
+                },
                 {
                   key: "JSON Schema",
                   data: [
                       {
                       value: "BINDINGNAME JSON Schema",
-                      href: "template.schema.json"
+                      href: "bindingname.schema.json"
```

Comment:
Isn't this the template file for the JSON Schema that goes in a binding? Do we need to change the name of this file if it doesn't refer to a "binding template"?

Reply:
Good catch... It should be reverted and the other references should reference https://github.com/w3c/wot-binding-templates/blob/main/bindings/protocols/template.schema.json instead

Reply:
Done